docs/outlook/add-and-remove-attachments-to-an-item-in-a-compose-form.md

```diff
@@ -158,6 +158,9 @@ The following APIs to get attachments in compose mode are available from [requir
 
 Use the [getAttachmentsAsync](/javascript/api/requirement-sets/outlook/preview-requirement-set/office.context.mailbox.item#methods) method to get the attachments of the message or appointment being composed.
 
+> [!NOTE]
+> In Outlook on the web and the new Outlook on Windows, users can select the **Upload and share** option to upload an attachment to OneDrive and include a link to the file in the mail item. However, since only a link is included, `getAttachmentsAsync` doesn't return this attachment.
```

Comment:
```suggestion
> In Outlook on the web and the new Outlook on Windows, users can select the **Upload and share** option to upload an attachment to OneDrive and include a link to the file in the mail item. However, since only a link is included, `getAttachmentsAsync` doesn't return this type of attachment.
```